mods/ts/sequences/misc.yaml

```diff
@@ -609,7 +609,7 @@ tracks16:
 		Offset: 0, -13, 42
 		ZRamp: 1
 		UseTilesetExtension: true
-		ShadowStart: 1
+		#ShadowStart: 1
```

Comment:
I think this needs at least a short comment explaining that this should only be re-enabled when rendering has been refactored enough to fix shadow from hiding actors.